sgminer.c

```diff
@@ -8769,7 +8771,7 @@ static void enable_curses_windows(void)
   getmaxyx(mainwin, y, x);
   statuswin = newwin(logstart, x, 0, 0);
   leaveok(statuswin, true);
-  logwin = newwin(y - logcursor, 0, logcursor, 0);
+  logwin = newwin(abs(y - logcursor), 0, logcursor, 0);
```

Comment:
abs or max(0, ...) ?

Reply:
https://linux.die.net/man/3/newwin
First parameter is nlines, it shouldn't be 0.